cli.go

```diff
@@ -967,21 +978,27 @@ func main() {
 						}
 					}
 
-					projectListKeys := c.StringSlice("list-key")
-					projectListNames := c.StringSlice("list")
-
-					if len(projectListNames) > 0 {
-						// Add project lists by name:
+					if len(projectListNames) > 0 || allListsFlag {
 						lists, err := client.ListProjectSelections()
 						if err != nil {
 							panic(err)
 						}
 
-						for _, name := range projectListNames {
-							list := lists.ByName(name)
-							if list == nil {
-								Warnf("List %q not found; skipping", name)
-							} else {
+						if len(projectListNames) > 0 {
```

Comment:
This check is kinda redundant; the loop won't iterate if the slice is empty.

```
						if len(projectListNames) > 0 {
```

Reply:
Right! Didn't realize it :O

Reply:
Done https://github.com/gagliardetto/lgtm-cli/pull/5/commits/7259ceca8bc0a9aec8247efbcf09de3fa880d5aa :P